### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Antidote/antidote-ios.localizablestrings](https://hosted.weblate.org/projects/antidote/antidote-ios-app/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: Antidote/zgh.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,404 @@
+
+
+/* Call screen text */
+"call_reaching" = "ⵉⵇⵇⴰⵔ...";
+/* Login screen text */
+"login_or_label" = "ⵏⵉⵖ";
+/* Login screen text */
+"create_account" = "ⵙⴽⵔ ⴽⵔⴰ ⵏ ⵓⵎⵉⴹⴰⵏ";
```
Comment (Tha14) [inline Antidote/zgh.lproj/Localizable.strings:8]: It seems that there is a problem here.

**Review by Tha14 — APPROVED** [review]

**Comment by zoff99** [comment]
@Tha14 do we need to add it to the project with xcode, so that it's avaiable in the app?

**Comment by zoff99** [comment]
like we did here: https://github.com/Zoxcore/Antidote/commit/1771279e692552c369693b16e052322c8429c46d

**Comment by Tha14** [comment]
> @Tha14 do we need to add it to the project with xcode, so that it's avaiable in the app?

Yes, as far as I know it's not enabled.

**Pull request merged**